src/Session/commands.py

```diff
@@ -1,14 +1,48 @@
 from src.Constants.commands_messages import *
 from src.Constants.constants import *
+from src.Food.food import Food
 from src.Session.profile import Profile
 from src.Utils.utils import listInQuotes, twoPrintOutcomes
 
 class Commands:
     def __init__(self):
         self.profile = None
 
+    ''' Prompts user to add foods to whichever food storage they are on. If the user is not in an opened 
+        food storage, suggests the user opens an existing food storage or create a new one. '''
     def add_food(self):
-        pass
+        # Profile check
+        if not self.profile:
+            print(NO_LOADED_PROFILE_MESSAGE, \
+                SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([CREATE_PROFILE, LOAD])))
+            return
+        
+        # Check if food storage is open
+        if self.profile.getOpenFoodStorage() != None:
+            food_list = []
+            while True: 
+                food_name = input(ADD_FOOD_NAME_MESSAGE)
+                expiration_date = input(ADD_FOOD_EXPIRATION_DATE_MESSAGE)
+                try:
+                    use_by_date = int(input(ADD_FOOD_USE_BY_DATE_MESSAGE))
```

Comment:
same thing as the other comment, if I just type in any non-number string here the int function will error out right?

Reply:
This case does have error handling. If you type a non-number, when python tries to cast it to an int, it'll error and go the 'except' block, where it creates Food object with the default `use_by_date`, which is `None`